index.md

```diff
@@ -108,6 +108,9 @@ Base Tools
 -	[PHPStorm code style](https://github.com/Automattic/PhpStorm-Resources)
 -	[Brackets](https://github.com/Tusko/brackets-wp-hint/)
 -	[Atom](https://atom.io/)
+- [WordPress Snippet](https://marketplace.visualstudio.com/items?itemName=tungvn.wordpress-snippet) - an extension for Visual Studio Code
```

Comment:
The latest commit there was on 02.02.2017 and the supported WP version is 4.6.1, while the latest/current release is 4.9.4. Adding an already outdated project while the main feature is supporting WordPress is something we would like to avoid.